doc/source/whatsnew/v0.19.0.rst

```diff
@@ -319,15 +313,15 @@ These provide date offsets anchored (by default) to the 15th and end of month, a
 
 .. ipython:: python
 
-    Timestamp('2016-01-01') + SemiMonthEnd()
+    pd.Timestamp('2016-01-01') + SemiMonthEnd()
```

Comment:
Is it possible to use `pd.offsets.SemiMonthEnd()`?
I find it more descriptive as the reader doesn't have to scroll around to find where `SemiMonthEnd()` came from.

Reply:
Do you mean `pd.tseries.offsets.SemiMonthEnd()`?   I think `SemiMonthEnd()` and `SemiMonthBegin()` are used right after importing from `pandas.tseries.offsets` and that was the only time they were used.  I could be wrong. Let me know if it is the case.

Reply:
`pd.tseries.offsets` is imported as `pd.offsets`. They are basically the same. But maybe it's better to leave it like `pd.tseries.offsets.SemiMonthEnd()` as I don't know whether it's been the case in `v0.19.0`

Reply:
The [document](https://pandas.pydata.org/pandas-docs/version/0.19/timeseries.html) for v0.19 also uses a single class name like `BDay` or `DataOffset`. So I will keep it as it is now for consistency.